JapaneseFilter/sections/css_extended.txt

```diff
@@ -290,6 +290,7 @@ j-cast.com#?#div.title-sec:contains(姉妹サイト)
 qiita.com#?##home-side-ad:upward(1)
 qiita.com#?##personalized-feed-side-ad:upward(1)
 qiita.com#?#.p-items_main > div[class^="css-"]:has(div[id^="dfp-slot-"])
+qiita.com#?#.p-items_main > div > div[class^="css-"]:has(> div > div[class^="css-"] > div[class^="css-"] > div[data-testid="ArticleAdsAfterBody_AdUnit"])
```

Comment:
Better `#?#.p-items_main > div > div[class^="css-"]:has(> div > div[class^="css-"] > div[class^="css-"] > div.ad-336x280)`. As long as good as a selector (true in this case), `##.class` or `###id` is faster and preferred to `##[attribute="something"]`.

Reply:
I see. Thank you.